lib/streamlit/web/server/routes.py

```diff
@@ -204,27 +204,6 @@ async def get(self) -> None:
         # disallows writing lists directly into responses due to potential XSS
         # vulnerabilities.
         # See https://www.tornadoweb.org/en/stable/web.html#tornado.web.RequestHandler.write
-        self.write(
```

Comment:
nit: You can remove the comment above this function since we anyways have to use a dict here to support all the other options. So there isn't even an option to do it differently.